rust/src/wallet.rs

```diff
@@ -156,6 +161,43 @@ impl Wallet {
         Ok(address.address)
     }
 
+    pub fn send_to_address(&self, address: Address, amount: u64) -> Result<Txid> {
```

Comment:
I implemented basically the same in https://github.com/itchysats/10101/pull/225 but I am estimating the fee using electrum client (hope that works correctly).
We will have to align this a bit I guess :)